pymongo/read_preferences.py

```diff
@@ -424,6 +424,45 @@ def __call__(self, selection):
                 self.max_staleness, selection))
 
 
+class _AggWritePref:
+    """Agg $out/$merge write preference.
+
+    * If there are no readable servers, or there is any pre-5.0 server, use
```

Comment:
Aren't we using Primary() in the first case and `pref` in the second?

Reply:
Yeah good catch. Fixed.